kubectl-plugin/README.md

```diff
@@ -44,3 +44,34 @@ Kubectl plugin/extension for Kuberay CLI that provides the ability to manage ray
         --tls-server-name string         Server name to use for server certificate validation. If it is not provided, the hostname used to contact the server is used
         --token string                   Bearer token for authentication to the API server
         --user string                    The name of the kubeconfig user to use
+
+### Retrieve Ray Cluster Head Logs
+
+    Usage:
+    log (RAY_CLUSTER_NAME) --out-dir [directory] [flags]
+
+    Aliases:
+    log, logs
+
+    Flags:
+        --as string                      Username to impersonate for the operation. User could be a regular user or a service account in a namespace.
+        --as-group stringArray           Group to impersonate for the operation, this flag can be repeated to specify multiple groups.
+        --as-uid string                  UID to impersonate for the operation.
+        --cache-dir string               Default cache directory (default "/Users/aaronliang/.kube/cache")
+        --certificate-authority string   Path to a cert file for the certificate authority
+        --client-certificate string      Path to a client certificate file for TLS
+        --client-key string              Path to a client key file for TLS
+        --cluster string                 The name of the kubeconfig cluster to use
+        --context string                 The name of the kubeconfig context to use
+        --disable-compression            If true, opt-out of response compression for all requests to the server
+    -h, --help                           help for log
+        --insecure-skip-tls-verify       If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure
+        --kubeconfig string              Path to the kubeconfig file to use for CLI requests.
+    -n, --namespace string               If present, the namespace scope for this CLI request
+        --out-dir string                 File Directory PATH of where to download the file logs to.
+        --request-timeout string         The length of time to wait before giving up on a single server request. Non-zero values should contain a corresponding time unit (e.g. 1s, 2m, 3h). A value of zero means don't timeout requests. (default "0")
+    -s, --server string                  The address and port of the Kubernetes API server
+        --tls-server-name string         Server name to use for server certificate validation. If it is not provided, the hostname used to contact the server is used
+        --token string                   Bearer token for authentication to the API server
+        --user string                    The name of the kubeconfig user to use
```

Comment:
I think we don't need to put help messages in README.md. Users can always get the help message with the `--help` flag. Putting help messages in README may cause out of sync between the documentation and the real implementation.